boards/arm/nucleo_f030r8/doc/nucleof334r8.rst

```diff
@@ -0,0 +1,181 @@
+    - Virtual COM port
+    - Mass storage
+    - Debug port
+- Support of wide choice of Integrated Development Environments (IDEs) including:
+    - IAR
+    - ARM Keil
+    - GCC-based IDEs
+
+More information about STM32F030R8 can be found here:
+       - `STM32F030 reference manual`_
+
+
+Supported Features
+==================
+
+The Zephyr nucleo_f030r8 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| IWDG      | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported in this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_f030r8/nucleo_f030r8_defconfig``
+
+Connections and IOs
+===================
+
+Each of the GPIO pins can be configured by software as output (push-pull or open-drain), as
+input (with or without pull-up or pull-down), or as peripheral alternate function. Most of the
+GPIO pins are shared with digital or analog alternate functions. All GPIOs are high current
+capable except for analog inputs.
+
+Board connectors:
+-----------------
+.. image:: img/nucleo_f030r8_connectors.png
+     :width: 800px
+     :align: center
+     :height: 619px
+     :alt: Nucleo F030R8 connectors
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_1_TX : PA9
+- UART_1_RX : PA10
+- UART_2_TX : PA2
+- UART_2_RX : PA3
+
+For mode details please refer to `STM32 Nucleo-64 board User Manual`_.
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+Nucleo F030R8 board includes an ST-LINK/V2-1 embedded debug tool interface.
+This interface is supported by the openocd version included in Zephyr SDK.
+
+Flashing an application to Nucleo F030R8
+----------------------------------------
+
+The sample application `blinky` is being used in this tutorial:
+
```

Comment:
I'd recommend deleting this code block (and change the colon ":" on the previous line to end with a period.